src/terminal/adapter/ut_adapter/inputTest.cpp

```diff
@@ -300,10 +301,62 @@ void InputTest::TerminalInputTests()
     inputEvent = IInputEvent::Create(irUnhandled);
     VERIFY_ARE_EQUAL(false, pInput->HandleKey(inputEvent.get()), L"Verify MENU_EVENT was NOT handled.");
 
-    Log::Comment(L"Testing FOCUS_EVENT");
-    irUnhandled.EventType = FOCUS_EVENT;
-    inputEvent = IInputEvent::Create(irUnhandled);
-    VERIFY_ARE_EQUAL(false, pInput->HandleKey(inputEvent.get()), L"Verify FOCUS_EVENT was NOT handled.");
+    Log::Comment(L"Testing FOCUS_EVENTs is handled by TestFocusEvents");

```

Comment:
This seems more like noise than signal -- it'll print up in every test run, _moments before TestFocusEvents runs_